### PR TITLE
add AdonisJs example

### PR DESCRIPTION
**Pull request by thetre97**

### CODE REVIEW

**Comment by thetre97** [comment]
Ah... Is this managed by the Zeit team?

**Comment by thetre97** [comment]
Example app running here - https://adonis-jmfuapcjas.now.sh/

File: README.md [hunk 1]
```diff
@@ -52,6 +52,7 @@ These deployments:
 | [misc-socket.io-chat](https://misc-socketio-chat-v2.now.sh) | [Node.js](https://en.wikipedia.org/wiki/Node.js) | The socket.io [chat demo](https://socket.io/demos/chat/) | 20.2M |
 | [misc-socket.io-whiteboard](https://misc-socketio-whiteboard-v2.now.sh) | [Node.js](https://en.wikipedia.org/wiki/Node.js) | The socket.io [whiteboard demo](https://socket.io/demos/whiteboard/) | 20.2M |
 | [misc-web-tty](https://misc-web-tty-v2.now.sh) | [Go](https://en.wikipedia.org/wiki/Go_(programming_language)) | A fully functional terminal in the browser powered by [gotty](https://github.com/yudai/gotty) | 6.1M |
+| [node-adonisjs](https://adonis-jmfuapcjas.now.sh/) | [Node.js](https://en.wikipedia.org/wiki/Node.js) | A minimal AdonisJs application |
```
Comment (TooTallNate) [inline README.md]: The image size is missing (it gets printed at the end of the "build" step). Also if you could keep the URL format the same with the example name + `-v2.now.sh` and we will take care of aliasing a deployment to there.

File: node-adonisjs/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,10 @@
+FROM node:alpine
```
Comment (TooTallNate) [inline node-adonisjs/Dockerfile]: We should use a multi-stage docker build here to optimize the file size (i.e. remove npm / yarn after installation):

```Dockerfile
FROM mhart/alpine-node:10 as build
WORKDIR /usr/src
COPY package.json ./
RUN yarn --production
COPY . .

FROM mhart/alpine-node:base-10
WORKDIR /usr/src
ENV PATH="./node_modules/.bin:$PATH"
ENV NODE_ENV="production"
ENV ENV_SILENT="true"
COPY --from=build /usr/src .
EXPOSE 3333
CMD ["node", "server.js"]
```

File: README.md (hunk 1 above)
Comment (thetre97) [reply README.md]: ah okay - done.

**Comment by TooTallNate** [comment]
Thank you @thetre97!

**Comment by TooTallNate** [comment]
Hey @thetre97. Our team was missing the required secret. Added one, and the alias is live now.

**Pull request merged**